package/MDAnalysis/visualization/streamlines.py

```diff
@@ -176,7 +176,7 @@ def produce_list_centroids_this_frame(list_indices_in_polygon):
         list_centroids_this_frame = []
         for indices in list_indices_in_polygon:
             if not indices[0].size > 0:  # if there are no particles of interest in this particular square
-                list_centroids_this_frame.append('empty')
```

Comment:
Same issue as per streamlines_3D, also can't get coverage over it (even though the new test assigns None to everything (hence the zeroed array return)